Src/Base/AMReX_ParmParse.cpp

```diff
@@ -1854,7 +1854,7 @@ int
 ParmParse::remove (const char* name)
 {
     auto const pname = prefixedName(name);
```

Comment:
Interesting that this did not throw an unused variable warning. Should we double check CI covers this?